src/compiler/emitter.ts

```diff
@@ -9,9 +9,9 @@ namespace ts {
 
     // Flags enum to track count of temp variables and a few dedicated names
     const enum TempFlags {
-        Auto      = 0x00000000,  // No preferred name
+        Auto = 0x00000000,  // No preferred name
         CountMask = 0x0FFFFFFF,  // Temp variable counter
-        _i        = 0x10000000,  // Use/preference flag for '_i'
+        _i = 0x10000000,  // Use/preference flag for '_i'

```

Comment:
I think it depends on the ruleset. I don't have any alignment rules turned on and I'm not sure how they'll work when they conflict with things like whitespace rules (which is presumably the issue here). I'll just preserve the alignment for now since we're so far from 0 TSLint errors.